### PR TITLE
feat: ios inspectable on webview2

### PR DESCRIPTION
**Pull request by ajpinedam**
GitHub Issue (If applicable): 

related to https://github.com/unoplatform/uno/discussions/18041

## PR Type

What kind of change does this PR introduce?

- Bugfix

## What is the new behavior?

Developers can use a feature flag to enable the `Inspectable` feature on WebView2 for iOS and Catalyst apps.

## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] Docs have been added/updated which fit [documentation template](https://github.com/unoplatform/uno/blob/master/doc/.feature-template.md) (for bug fixes / features)
- [ ] [Unit Tests and/or UI Tests](https://github.com/unoplatform/uno/blob/master/doc/articles/uno-development/working-with-the-samples-apps.md) for the changes have been added (for bug fixes / features) (if applicable)
- [ ] Validated PR `Screenshots Compare Test Run` results.
- [x] Contains **NO** breaking changes
- [x] Associated with an issue (GitHub or internal) and uses the [automatic close keywords](https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue).
- [x] Commits must be following the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/#summary) specification.

## Other information

Internal Issue (If applicable):

### CODE REVIEW

**Review by MartinZikmund — APPROVED** [review]

File: src/Uno.UI/FeatureConfiguration.cs [hunk 1]
```diff
@@ -703,6 +703,19 @@ public static class WebView
...
+		public static class WebView2
+		{
+#if __IOS__
+			/// <summary>
+			/// Sets wether the <see cref="WebView2"/> object is inspectable or not.
+			/// </summary>
+			/// <remarks>
+			/// On iOS and Catalyst this means that developers can use the Safari Web Developers tools to debug apps with <see cref="WebView2"/>
+			/// </remarks>
+			public static bool IsInspectable { get; set; };
```
Comment (MartinZikmund) [inline src/Uno.UI/FeatureConfiguration.cs:715]: Please add docs for the new flag as well

Comment (ajpinedam) [reply src/Uno.UI/FeatureConfiguration.cs:715]: Working on that this morning.

Comment (ajpinedam) [reply src/Uno.UI/FeatureConfiguration.cs:715]: added :)

File: doc/articles/controls/WebView.md [hunk 2]
```diff
@@ -127,3 +127,17 @@ The web files can reference each other in a relative path fashion, for example,
...
+
+## iOS and macOS (Catalyst) specifics
+
+From MacOS, inspecting applications using `WebView2` controls using the Safari Developer Tools is possible. [Here's](https://developer.apple.com/documentation/safari-developer-tools/inspecting-ios) a detailed guide on how to do it. To make this work, enable this feature in your app by adding the following capabilities in your `App.Xaml.cs`:
+
+```csharp
+public App()
+{
+    this.InitializeComponent();
+#if __IOS__
```
[the remark this answers is not in the document]
Comment (ajpinedam) [reply doc/articles/controls/WebView.md:139]: Completed

**Review by jeromelaban — APPROVED** [review]

**Comment by jeromelaban** [comment]
@mergifyio backport release/stable/5.4

**Pull request merged**